hooks/hiero_get_shot.py

```diff
@@ -20,14 +20,22 @@ def execute(self, task, item, data, **kwargs):
         Takes a hiero.core.TrackItem as input and returns a data dictionary for
         the shot to update the cut info for.
         """
-        # get the parent sequence for the Shot
-        sequence = self._get_sequence(item, data)
+
+        # get the parent entity for the Shot
+        parent = self.parent.execute_hook(
+            "hook_get_shot_parent",
+            hiero_sequence=item.parentSequence(),
+            data=data
+        )
```

Comment:
That sounds right.  I was thinking about the fact that anybody who overrode the parenting behavior would need to update their hook anyway to get correct Cut parenting... so we couldn't be 100% backward compatible (or at least have the Cuts use the overridden hook in any intelligent way).  This reflects that and sounds very sane to me.

Thanks!